libopenage/renderer/opengl/texture.cpp

```diff
@@ -56,7 +56,7 @@ GlTexture::GlTexture(const resources::TextureData& data)
 	// TODO these are outdated, use sampler settings
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
-
+	glBindTexture(GL_TEXTURE_2D, 0);
```

Comment:
This is unnecessary - we don't care which texture is bound as long as we always remember to bind ones we want to use when we need them. Having them bound while they're not being used does not hurt.

Reply:
I just thought it was a good practice and does not affect performance.

Reply:
While I can't find a source for this right now, redundant state changes like this _can_ be bad. A single one in the constructor is not important, but as a general rule it's better to avoid them.

Reply:
Sure. This does not affect the program in anyway